catalog/transformer/catalog-transformer-geojson-input/src/test/java/ddf/catalog/transformer/input/geojson/GeoJsonExtensibleTest.java

```diff
@@ -481,6 +483,47 @@ public void testBasicMetacardTypeNoMetacardType()
     verifyBasics(metacard);
   }
 
+  @Test
+  public void testTransformNoId() throws IOException, CatalogTransformerException {
+    Metacard metacard =
+        transformer.transform(new ByteArrayInputStream(sampleBasicMetacard().getBytes()));
+
+    assertEquals("ddf.metacard", metacard.getMetacardType().getName());
```

Comment:
✏️ Consider writing this assertion (and the other `assertEquals` throughout these changes) in the `assertThat([value], [matcher statement])` format for improved readability.